src/vos/vos_obj_index.c

```diff
@@ -282,8 +282,8 @@ vos_oi_find_alloc(struct vos_container *cont, daos_unit_oid_t oid,
 	if (rc != 0)
 		return rc;
 
-	rc = ilog_update(loh, NULL, epoch, dth != NULL ? dth->dth_op_seq : 1,
-			 false);
+	rc = ilog_update(loh, NULL, epoch,
+			 dtx_is_valid_handle(dth) ? dth->dth_op_seq : 1, false);
```

Comment:
this will conflict with my patch since I switched to using MAX rather than 1.  The race is on I guess.

Reply:
Sorry for trouble you to resolve the merge conflict.